File: app/src/main/java/altcoin/br/decred/utils/Bitcoin.java
package altcoin.br.decred.utils;

import com.android.volley.Response;

public class Bitcoin {

    public static void convertBtcToBrl(Response.Listener<String> listener) {
        InternetRequests internetRequests = new InternetRequests();

        String url = "http://api.bitvalor.com/v1/ticker.json";

        internetRequests.executeGet(url, listener);
    }

    public static void convertBtcToUsd(Response.Listener<String> listener) {
        InternetRequests internetRequests = new InternetRequests();

        String url = "https://www.bitstamp.net/api/v2/ticker/btcusd/";

        internetRequests.executeGet(url, listener);
    }

}

File: app/src/main/java/altcoin/br/decred/utils/Bitcoin.kt
package altcoin.br.decred.utils

import com.android.volley.Response

object Bitcoin {
    fun convertBtcToBrl(listener: Response.Listener<String>) {
        val internetRequests = InternetRequests()

        val url = "https://api.blinktrade.com/api/v1/BRL/ticker"

        internetRequests.executeGet(url, listener)
    }

    fun convertBtcToUsd(listener: Response.Listener<String>) {
        val internetRequests = InternetRequests()

        val url = "https://www.bitstamp.net/api/v2/ticker/btcusd/"

        internetRequests.executeGet(url, listener)
    }

}
